src/internal/sdata/sdata.go

```diff
@@ -108,11 +108,31 @@ func Copy(w TupleWriter, r TupleReader, row Tuple) (n int, _ error) {
 	return n, nil
 }
 
+func NewTupleFromTableInfo(info *pachsql.TableInfo) (Tuple, error) {
+	tuple := make(Tuple, len(info.Columns))
+	for i, ci := range info.Columns {
+		var err error
+		tuple[i], err = makeTupleElement(ci.DataType, ci.IsNullable)
+		if err != nil {
+			return nil, err
+		}
+	}
+	return tuple, nil
+}
+
 func NewTupleFromTable(ctx context.Context, db *pachsql.DB, tableName string) (Tuple, error) {
 	info, err := pachsql.GetTableInfo(ctx, db, tableName)
 	if err != nil {
 		return nil, err
 	}
+	return NewTupleFromTableInfo(info)
+}
+
+func NewTupleFromTableTx(ctx context.Context, tx *pachsql.Tx, tableName string) (Tuple, error) {
```

Comment:
this and `NewTupleFromTable` are still unused, right? To be honest, now that they are just two function calls maybe we don't need them